AdvancedDLSupport/NativeLibraryBase.cs

```diff
@@ -106,6 +110,22 @@ protected void ThrowIfDisposed()
             }
         }
 
+        /// <summary>
+        /// Adds a delegate to keep it's lifetime till this library gets disposed.
```

Comment:
Use "until" instead of "till".

Reply:
is that some standard to use "until" here?(just curious)

Reply:
In general, I prefer to keep documentation fairly formal - that's about the only reason.

Reply:
yeah didn't btw know that "until" is more formal(until I googled it after your comment about using "until" :D)